WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -670,3 +534,31 @@ def stats(request):
     }
 
     return context_dictionary
+
+
+def started_by_id_to_name(started_by_id=None):
+    """
+    Returns name of the user or team that submitted an autoreduction run given an ID number
+    :param started_by_id: The ID of the user who started the run, or a control code if not started by a user
+    :return:
+        If started by a valid user, returns the name either of the user in the format '[forename] [surname]'.
+        If started automatically, returns "Autoreduciton service".
+        If started manually, returns "Development team".
+        Otherwise, returns None.
+    """
+    name = None
+    if started_by_id is not None:
+        if started_by_id == -1:
```

Comment:
Potentially out of scope for this issue, but these feel like magic numbers to me. But it would be nice to have development team and service IDs as constants somewhere as well as the names.

Reply:
I'll take a look at that. This PR would be a good opportunity to do some more extensive refactoring and cleanup.

Reply:
I remember the origin of this. Since a user id cannot be <1, 0 and -1 were used as sort of 'control codes' to indicate a system submission or manual submission respectively.

I agree it would be better to refactor into constants